src/query/list_file.c

```diff
@@ -5686,64 +5542,44 @@ qfile_lookup_list_cache_entry (THREAD_ENTRY * thread_p, int list_ht_no, const DB
   /* look up the hash table with the key */
   lent = (QFILE_LIST_CACHE_ENTRY *) mht_get (qfile_List_cache.list_hts[list_ht_no], params);
   qfile_List_cache.lookup_counter++;	/* counter */
+
   if (lent)
     {
+      unsigned int i;
+
+#if defined(SERVER_MODE)
```

Comment:
The cache is deleted only for server mode?

Reply:
Ok, I will fix it.